ledger/ledger-api-test-tool-on-canton/BUILD.bazel

```
@@ -82,6 +82,7 @@ conformance_test(
         ",ParticipantPruningIT" +  # see "conformance-test-participant-pruning" below
         ",ConfigManagementServiceIT,LedgerConfigurationServiceIT" +  # dynamic config management not supported by Canton
         ",ClosedWorldIT" +  # Canton currently fails this test with a different error (missing namespace in "unallocated" party id)
+        ",CommandServiceIT,CommandSubmissionCompletionI" + # FIXME: canton should be updated with last SDK to produce proper output message for those tests to pass
```

```suggestion
        ",CommandServiceIT,CommandSubmissionCompletionIT" + # FIXME: canton should be updated with last SDK to produce proper output message for those tests to pass
```
you will also need to fix the compat tests